src/frameworks/drupal8/solr.md

```diff
@@ -40,9 +40,7 @@ solrsearch:
     configuration:
         cores:
             maincore:
-                core_config: !include
-                    type: archive
-                    path: "solr-conf/6.x"
+                conf_dir: !archive "solr-conf/6.x"
```

Comment:
"solr-conf/8.x" ?

Reply:
I'm not sure about changing the Drupal page yet, as there's some weirdness with the newer Solr bridge I've not sorted out yet.  See #1077.  I'd rather not mess with the version examples there until we know what else needs to change.